### PR TITLE
Filter

### PR DESCRIPTION
**Pull request by FatimaAbdimalik**
Thank you for contributing! To help the review process, please provide the following:

### Proposal
Provide a short description of the change you are proposing and the reasons for it.

### Related
Provide links to any associated pull requests or issues. You can use `#123` to link to a PR or issue in this repository, or `user/repo#123` to link to other repositories.

### Checklist

- [ ] I have read the [code of conduct] and [contributing guide]
- [ ] I have made this pull request to the `master` branch
- [ ] I have run all of the automated validation using `npm run ship`
- [ ] I have added myself to the `"contributors"` list in the `package.json` (or do not want to)

[code of conduct]: https://github.com/textbook/starter-kit/blob/master/.github/CODE_OF_CONDUCT.md
[contributing guide]: https://github.com/textbook/starter-kit/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: client/src/components/Filter.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        style={{
+          width: "20rem",
+          backgroundColor: "white",
+          marginLeft: "12rem",
+          color: "black",
+        }}
+        onChange={handleChange}
+      />
+      {/* <StudentSubmission id={id} /> */}
+      {/* <SubmissionCard id={id} /> */}
```
Comment (Hedyeh65) [inline client/src/components/Filter.js:29]: please remove this lines

File: client/src/components/LessonsDropDown.js [hunk 2]
```diff
@@ -2,9 +2,9 @@ import axios from "axios";
...
 import SubmitWork from "./SubmitWork";
 
-const LessonsDropDown = ({ module }) => {
+const LessonsDropDown = ({ module, id }) => {
   const [lesson, setLesson] = useState();
-
+  const [lessonValue, setLessonValue] = useState();
   useEffect(() => {
     axios.get("http://localhost:3100/api/syllabus/lessons").then((response) => {
       console.log(response);
```
Comment (Hedyeh65) [inline client/src/components/LessonsDropDown.js:10]: This console.log should be removed

File: client/src/components/MentorViewSubmission.js [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+    value.push([e.target.id, e.target.value]);
+    setValue(value);
+  };
+
+  const handleInputChange = (e) => {
+    return setSearchItem(e.target.value);
+  };
+
+  const handleSubmitFeedback = (e) => {
+    console.log(value);
```
Comment (Hedyeh65) [inline client/src/components/MentorViewSubmission.js:30]: please remove this console.log

File: client/src/components/MentorViewSubmission.js [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+      .catch((err) => console.log(err));
+  }, [student_id]);
+
+  useEffect(() => {
+    if (cardData !== undefined) {
+      const foundTiles = cardData.filter((p) =>
+        p.title.toLowerCase().includes(searchItem.toLowerCase())
+      );
+      setSearchResult(foundTiles);
+      console.log(cardData);
```
Comment (Hedyeh65) [inline client/src/components/MentorViewSubmission.js:74]: please remove these 2 console.logs

File: client/src/components/MentorViewSubmission.js [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+              <Card.Title style={{ width: "40rem", display: "flex" }}>
+                <Accordion.Toggle as={Button} variant="light" eventKey="0">
+                  {" "}
+                  {card.title}
+                </Accordion.Toggle>
+              </Card.Title>
+              <Accordion.Collapse eventKey="0">
+                <div className="card-color">
+                  <div id="card-date">
+                    {" "}
```
Comment (Hedyeh65) [inline client/src/components/MentorViewSubmission.js:199]: please remove the brackets

File: client/src/components/ModuleDropDown.js [hunk 6]
```diff
@@ -1,14 +1,14 @@
 import React, { useState, useEffect } from "react";
 import axios from "axios";
 import LessonsDropDown from "./LessonsDropDown";
-const ModuleDropDown = () => {
+const ModuleDropDown = ({ id }) => {
   const [subject, setSubject] = useState();
   const [select, setSelect] = useState();
-
+  console.log(id);
```
Comment (Hedyeh65) [inline client/src/components/ModuleDropDown.js:7]: pleaswe remove this log

File: client/src/components/SubmissionCard.js [hunk 7]
```diff
@@ -1,22 +1,192 @@
...
+      .catch((err) => console.log(err));
+  }, []);
+
+  useEffect(() => {
+    if (cardData !== undefined) {
+      const foundTiles = cardData.filter((p) =>
+        p.title.toLowerCase().includes(searchItem.toLowerCase())
+      );
+      setSearchResult(foundTiles);
+      console.log(cardData);
```
Comment (Hedyeh65) [inline client/src/components/SubmissionCard.js:26]: remove these 2 console.logs

File: client/src/components/SubmissionCard.js [hunk 8]
```diff
@@ -1,22 +1,192 @@
...
+        }}
+        onChange={handleChange}
+      />
+      {cardData.map((card, index) => {
+        return (
+          <Accordion>
+            <Card className="submission-card" key={index}>
+              <Card.Title style={{ width: "40rem", display: "flex" }}>
+                <Accordion.Toggle as={Button} variant="light" eventKey="0">
+                  {" "}
```
Comment (Hedyeh65) [inline client/src/components/SubmissionCard.js:75]: remove these brackets

**Review by Hedyeh65 — CHANGES_REQUESTED** [review]
good job Fatima!

File: client/src/components/Filter.js [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+          marginLeft: "12rem",
+          color: "black",
+        }}
+        onChange={handleChange}
+      />
+    </div>
+  );
+};
+
+export default Filter;
```
Comment (SULAIMAN-100) [inline client/src/components/Filter.js:32]: could you give the file more specified name like filter cards or filter card title.

**Review by SULAIMAN-100 — APPROVED** [review]

**Review by Hedyeh65 — APPROVED** [review]

**Pull request merged**